packages/client-react-streaming/src/transportedQueryRef.ts

```diff
@@ -211,32 +267,23 @@ export function isTransportedQueryRef(
 }
 
 /** @public */
+/*
```

Comment:
```suggestion
/**
```

Was this meant as a doc block or general comment? If general comment, mind if I ask that we use `//` on each line so that its obvious this isn't a typo?

Reply:
Intentionally comment, not docblock. Changing to `//`.